Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1172,6 +1172,35 @@ std::string getPrimitiveName(
   }
   return name;
 }
+
+/// Helper used to log only once per unsupported primitive mode.
+struct PrimModeLogHelper {
+  std::array<std::atomic_bool, (size_t)MeshPrimitive::Mode::TRIANGLE_FAN + 1>
```

Comment:
When testing this locally, I needed this change to make it compile:

```suggestion
  std::array<std::atomic_bool, (size_t)CesiumGltf::MeshPrimitive::Mode::TRIANGLE_FAN + 1>
```